src/tests/BatchReplayRendererTest.cpp

```diff
@@ -182,7 +182,7 @@ void BatchReplayRendererTest::testIntegration() {
       renderer->setSensorTransformsFromKeyframe(envIndex, userPrefix);
     }
 
-    renderer->render(imageViews);
+    renderer->render(imageViews, nullptr);
```

Comment:
Hmm, probably would be good to test all combinations here (color only, depth only, color + depth) so there's no accidentally unhandled / crashing case.

For verifying the depth, in the batch renderer tests themselves I'm only checking if the values make sense (being 0.999something, not 0 and not 1 in a few arbitrarily chosen pixels). Should be enough I think, no need to do any image comparison.